File: Algoritmos e logica de Programacao/C/tipo_de_dados_e_variaveis.c
#include <stdio.h>
#include <string.h>

/*É possivel incluir atribuição na declaração da variavél _ Exemplo (int idade = 20) e (char nome[50] = "Maria Silva") */


int main()
{

    int idade;
    double salario, altura;
    char genero;
    char nome[50];

    idade = 20;
    salario = 5800.5;
    altura = 1.63;
    genero = 'F';
    strcpy(nome, "Maria Silva");

    printf("IDADE = %d\n", idade);
    printf("SALARIO = %1f\n", salario);
    printf("ALTURA = %1f\n", altura);
    printf("GENERO = %c\n", genero);
    printf("NOME = %s\n", nome);

    return 0;

}

File: Algoritmos e logica de Programacao/Java/tipo_de_dados_e_variaveis.java

/* variaveis e tipo de dados em Java */
/* Exemplo da aula */


	import java.util.Locale; 
	public class tipo_de_dados_e_variaveis {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		
		int idade;
		double salario, altura;
		char genero;
		String nome;
		
		idade = 30;
		salario = 1100.5;
		genero = 'F';
		nome = "Maria Jose";
		altura = 1.60;
		
		System.out.println("Idade = " + idade);
		System.out.println("Salario = " + String.format("%.2f", salario));
		System.out.println("Altura = " + String.format("%.2f", altura));
		System.out.println("Genero = " + genero);
	    System.out.println("Nome = " + nome); 
	    
	 } 
	
	}
	